### ISSUE TITLE
Role aliases for common aliases (new syntax)

### ISSUE THREAD

**Issue by OdyX** [sephii/taxi-zebra#14]
Some activities (such as `_internal`) are bound to always link to different roles; caching local role defaults doesn't make a lot of sense.

I suggest a new optional syntax, with role aliases too (Zebra, I'm staring at you):

```
08/01/2019

# activity    (optional) role    time       description
infra_daily   internal_it_ll     0945-1030  Role-filling publicity and handling for Security Helper role in LS and BE.
```

This would impose having a whole structure for role aliases, in Zebra too.

BUT… I think it would really make a lot of sense to think about roles at timesheet-filling time, and not at `taxi ci` time.

**Comment by sitron** [comment]
i agree. Not only it makes more sense to do it at timesheet-filling time, it would also be way less time consuming when you have to push 10 entries for a day imo.
Note that for the alias you could use a custom field directly in Holaspirit (vs handling the alias management in Zebra)

**Comment by jeanmonod** [comment]
Not sure I agree on this one. Sometime we want to target an activity, sometime we want to target a role, and I guess in very few case we want to be able to express both at the same time. Having an optional column, will add a lot of complexity in the taxi file, and will be used for little case... What about keeping the same format as now, but allow to use activity aliases OR role aliases? Something like:

    _rawbot_ll     2      Tactical meeting             # Goes to internal stuff on role LeadLink Rawbot
    top_23         3.5    TOP-345 Review the story     # Goes to activity sprint 23 on Top project, with my Swing Dev role that I have configured at the first commit
    _fruit_lord    1      New contract negotiation     # Goes to internal stuff on role Lord of the fruits
    _internal      0.5    Emails and Slack             # Goes to internal stuff, role will be asked at commit

We can even imagine been able to define both in the same first col

    _iem_2019/iem_operator        0.5       Sending form to all   # Goes to IEM 2019 with Iem Operator role

**Comment by sephii** [comment]
Using separate aliases to map to different roles is the easiest way to implement it since that's already how it works. But I really like the `alias/role` syntax.

**Comment by jeanmonod** [comment]
Until we find a clean solution, maybe this could help: https://zebra.liip.ch/timesheets/taxi-aliases
